spec/feature/user_birth_record_spec.rb

```diff
@@ -51,14 +52,31 @@
 
             click_on 'Find'
 
+            # look for exactly 1 'add' button on a card for the target birth record
+            expect(page).to have_css(
+              ".birth-record[data-id='#{target_record.to_param}'] .birth-record__button[data-verb='add']",
+              count: 1
+            )
+
+            click_link 'add'
+
             expect(page).to have_selector('.card', count: 1)
-            expect(page).to have_css("##{target_record.model_name.param_key}--#{target_record.primary_key_string}")
```

Comment:
@lamorrison I'm not sure what this is looking for (other than it's a css class). Should we add something back here?